pyiron/base/job/core.py

```diff
@@ -801,6 +801,36 @@ def __del__(self):
         del self._master_id
         del self._status
 
+    def _create_groups_recursively(self, hdf, key, value):
```

Comment:
OK, now that I see the functions maybe it makes more sense to move them to `hdfio` . I am sorry for the confusion.